shell/src/main/java/alluxio/shell/command/CpCommand.java

```diff
@@ -566,11 +575,12 @@ private void copyFileToLocal(AlluxioURI srcPath, AlluxioURI dstPath)
         out.write(buf, 0, t);
         t = is.read(buf);
       }
-      if (!tmpDst.renameTo(dstFile)) {
+      if (!tmpDst.renameTo(outputFile)) {
         throw new IOException(
-            "Failed to rename " + tmpDst.getPath() + " to destination " + dstPath);
+            "Failed to rename " + tmpDst.getPath() + " to destination "
+                    + "file://" + outputFile.getPath());
       }
-      System.out.println("Copied " + srcPath + " to " + dstPath);
+      System.out.println("Copied " + srcPath + " to " + "file://" + outputFile.getPath());
```

Comment:
it might be better to stick with `dstPath` here so that we mirror the user's input, e.g. if they say `cp file dir` we say "Copied file to dir"  as opposed to "Copied file to dir/file".